src/status_im/ui/screens/chat/components/reply.cljs

```diff
@@ -89,7 +89,7 @@
      [rn/view {:style (styles/reply-content)}
       [icons/icon :main-icons/connector {:color (theme-colors quo2.colors/neutral-40 quo2.colors/neutral-60)
                                          :container-style {:position :absolute :left 10 :bottom -4 :width 16 :height 16}}]
-      [rn/view {:style {:position :absolute :left 34 :right 54 :top 3 :flex-direction :row :align-items :center :width "45%"}}
+      [rn/view {:style {:position :absolute :left 34 :top 3 :flex-direction :row :align-items :center :width "45%"}}
```

Comment:
its better to use fixed margins instead of percentages, percentage are not reliable, margins will be different on different devices if we use percentages

Reply:
@flexsurfer using right will not work here. Actually `right` has no effect here and can be removed. Text components normally overflow their parent. So, for `right` to have an effect `:overflow :hidden` must be set. But then, the text will just be cut without ellipsizing. 
For text components to ellipsize their parent need to have a width (afaik). `width 45%` will allow each of the two text components we have (user's name, and message text) to have a width of 45%, so total is 90%.

Reply:
cc @briansztamfater